src/components/multi_selection_list/multiselection_list.js

```diff
@@ -204,30 +204,35 @@ class MultiSelectionList extends PureComponent {
   }
 
   listFilter() {
-    const { searchPlaceholder, searchIcon } = this.props;
-
-    return (
-      <FormControl
-        className={classNames(
-          styles.list_filter_container,
-          styles.search_wrapper
-        )}
-        fullWidth
-      >
-        <Input
-          id="search"
-          value={this.state.searchTerm}
-          placeholder={searchPlaceholder}
-          onChange={this.onSearchTermChange}
-          className={styles.input}
-          endAdornment={
-            <InputAdornment position="end">
-              <Icon>{searchIcon}</Icon>
-            </InputAdornment>
-          }
-        />
-      </FormControl>
-    );
+    const { searchPlaceholder, searchIcon, searchComponent } = this.props;
+
+      return (
```

Comment:
What I'm suggesting is:
```
const SearchComponent = searchComponent;
return <SearchComponent />
```

in the default props define searchComponent as the <SeachComponent/> we use internally.